laravel-app/app/Http/Controllers/ProfileController.php

```diff
@@ -76,4 +77,14 @@ public function show(int $userId): RedirectResponse|View
             'likesCount' => $likesCount,
         ]);
     }
+
```

Comment:
add comment using docbloc without including @param and @return.

Reply:
Removing the **comments without param and return** creates conflicts with **larastan** sir

Reply:
fix it!

Reply:
done fixing! we used phpstan-ignore-line

Reply:
is this done?